idr0054-study.txt

```diff
@@ -37,7 +37,7 @@ Study License	CC-BY 4.0
 Study License URL	https://creativecommons.org/licenses/by/4.0/			
 Study Copyright	Durand et al.			
 Study Data Publisher	University of Dundee			
-Study Data DOI				
+Study Data DOI	10.17867/10000122			

```

Comment:
I think the study parser is currently expecting a DOI URL of form `https://doi.org/xxxx/yyyy`. It might be worth supporting this form in which case happy to get this merged and  fix `idr-metadata`accordingly.